.github/workflows/release-bumper.yml

```diff
@@ -45,19 +45,14 @@ jobs:
         with:
           go-version: '1.17' # The Go version to download (if necessary) and use.
 
-      - name: Check file existence
-        id: check_release_bumper
-        uses: andstor/file-existence-action@v1
-        with:
-          files: "automation/release-bumper/release-bumper.sh"
-
       - name: Check for new releases and update
         if: steps.check_release_bumper.outputs.files_exists == 'true'
         env:
           UPDATED_COMPONENT: ${{ github.event.inputs.component }}
           UPDATED_VERSION: ${{ github.event.inputs.version }}
         run: |
-          ./automation/release-bumper/release-bumper.sh
+          curl -s https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/main/automation/release-bumper/release-bumper.sh | bash
+          
```

Comment:
In older stabilization branches, the workflow is using older release-bumper script versions that do not contain newly added logic to only update .z releases